### PR TITLE
Improve some PHPDocs

### PR DESCRIPTION
**Pull request by Chouby**
All is in the title.

### CODE REVIEW

File: settings/table-string.php [hunk 1]
```diff
@@ -211,12 +211,12 @@ protected function search_in_translations( $mos, $s ) {
 	}
 
 	/**
-	 * Sort items
+	 * Sort registered string items.
```
Comment (Hug0-Drelon) [inline settings/table-string.php:214]:
```suggestion
	 * Sorts registered string items.
```

**Review by Hug0-Drelon — COMMENTED** [review]
https://github.com/polylang/polylang/blob/1efdc84691bbccddcfdf98f4cb53af5e806dcf07/include/olt-manager.php#L73 
https://github.com/polylang/polylang/blob/1efdc84691bbccddcfdf98f4cb53af5e806dcf07/integrations/integrations.php#L34
https://github.com/polylang/polylang/blob/1efdc84691bbccddcfdf98f4cb53af5e806dcf07/modules/wpml/wpml-config.php#L40

Maybe use `accesses` instead of `access` on these previous lines if it's not a noun.

**Comment by Chouby** [comment]
> Maybe use `accesses` instead of `access` on these previous lines if it's not a noun.
In this case the noun is ok for me.

**Review by Hug0-Drelon — APPROVED** [review]
OK 👍 
But the tests are failing, probably not because of this PR.

**Pull request merged**